initialize-repos.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+clone () {
+  git clone git@github.com:windbreaker-io/$1 repos/$1
+}
+
+mkdir repos
+
+set -e
+
+# clone projects into repos folder
+clone windbreaker
+clone rss-watcher
+clone npm-watcher
```

Comment:
This works for now, but in the future we'll probably want to pull updates if the repos have already been cloned previously.

Reply:
I agree

Reply:
I think this really is as simple as:

```
clone () {
  if cd repo; then git pull && cd ../; else git clone git@github.com:windbreaker-io/$1 repos/$1; fi
}
```

Reply:
+1 on the bash function, though `if [[ -d "repo/.git" ]]...` is how I would do it, I didn't actally know that you could `if` on cd.